nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/serialization/VersionedFlowSynchronizer.java

```diff
@@ -378,6 +378,7 @@ private void synchronizeFlow(final FlowController controller, final DataFlow exi
 
             if (versionedFlow != null) {
                 controller.setMaxTimerDrivenThreadCount(versionedFlow.getMaxTimerDrivenThreadCount());
+                controller.setMaxEventDrivenThreadCount(versionedFlow.getMaxTimerDrivenThreadCount());
```

Comment:
This should be:
controller.setMaxEventDrivenThreadCount(versionedFlow.getMax**Event**DrivenThreadCount());

Reply:
yeah.. what @markap14 said. I missed his comment.
Would love to see this quickly fixed so it can be added to 1.19.0. Thanks!